server/src/main/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthRequest.java

```diff
@@ -38,7 +39,7 @@ public class ClusterHealthRequest extends MasterNodeReadRequest<ClusterHealthReq
      * Only used by the high-level REST Client. Controls the details level of the health information returned.
      * The default value is 'cluster'.
      */
-    private Level level = Level.CLUSTER;
+    private String level = StatsLevel.CLUSTER.getName();
```

Comment:
This seems like a step in the wrong direction. The point of introducing an `enum` is so we can be _more_ strongly typed.